CHANGELOG.md

```diff
@@ -13,6 +13,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - Short option for `--accounts-file` flag has been removed.
 - Short option for `--contract-address` is now `-d` instead of `-a`.
+- Test performance has been optimized. When using a filter, only the matching tests are compiled. The summary no longer displays the number of filtered-out tests when running all tests. When a filter is applied, it now shows `other filtered out` rather than the previous `<n> filtered out`.
```

Comment:
This should be in forge section, not cast.

Also I'd reword it so it says that this optimized compilation only happens when filtering by name. Not in case you pass a `path::to::test` or `path::to::module::`

Reply:
I reworded it to 
```
Test performance has been optimized - using filter and `--exact` flag will result in compilation of only the selected test.```
because (as we discussed) it will be optimized only with `--exact` flag.